xAPI.md

```diff
@@ -770,16 +770,42 @@ The table below lists all properties of the Verb Object.
 	</tr>
 </table>
 
-###### Requirements
+###### Verb Id Requirements
 
-* The display property MUST be used to illustrate the meaning which is already determined by the Verb IRI.
 * A system reading a Statement MUST use the Verb IRI to infer meaning.
+* The IRI contained in an id SHOULD contain a human-readable portion which SHOULD provide meaning enough 
+to disambiguate it from other similar(in syntax) Verbs.
+* A single Verb IRI MUST NOT be used to refer to multiple meanings.
+
+###### Verb Display AP Requirements
+* The display property SHOULD be used by all Statements.
+* The display property MUST be used to illustrate the meaning which is already determined by the Verb IRI.
+
+###### Verb Display LRS Requirements
+The requirements below relate to the display property as returned by the LRS via the API.  
+
+* When queried for statements with a format of "exact", the LRS MUST return the Display property 
```

Comment:
I don't see how this can be a MUST. In general the LRS must return what was sent "statements are immutable", however we've previously determined that this concept applies to the value or meaning of properties within statements and not any particular serialization (eg: it's valid to change timestamp format as long as it still refers to the same point in time).

It's presumably based on this principle that the following SHOULD* requirements for modifying the display are OK -- however that same principle implies that there was never any implied "MUST" for the exact behavior above -- by these changes we're saying that according to the current spec the LRS can currently return anything it wants for verb display properties and we're filling in the ambiguity. Therefore we can't have any requirement associated with verb display stronger than a "SHOULD*"